app/utils/chainUtils.ts

```diff
@@ -12,6 +12,7 @@ export const CHAIN_DOCS_URL: KeyValuePair<string> = {
   "bsc-mainnet": "binance-smart-chain-api/intro",
   "bsc-archival": "binance-smart-chain-api/intro",
   "celo-mainnet": "celo-api/intro",
+  "celestia-archival": "celestia-api/endpoints/celestia-archival",
```

Comment:
probably need to add this mapping to the 3 others as well right?

Reply:
Yup, but their docs are still not added yet.